### PR TITLE
lynx: add page

### PR DESCRIPTION
**Pull request by alimalek71**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by alimalek71** [comment]
#4629

File: pages/common/lynx.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`lynx -use_mouse {{example.com}}`
+
+- Force color mode on, if available:
+
+`lynx -color {{example.com}}`
+
+- Specify a file to use to read/write cookies:
+
+`lynx -cookie_file {{example.com}}`
```
Comment (navarroaxel) [inline pages/common/lynx.md:24]: Should we add a `{{path/to/file}}`?

File: pages/common/lynx.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Turn on mouse support, if available:
+
+`lynx -use_mouse {{example.com}}`
+
+- Force color mode on, if available:
+
+`lynx -color {{example.com}}`
+
+- Specify a file to use to read/write cookies:
```
Comment (navarroaxel) [inline pages/common/lynx.md:22]:
```suggestion
- Open a link, using a specific file to read and write cookies:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @alimalek71 I left some comments.

**Comment by navarroaxel** [comment]
Looks like there is another PR for this command #4764, but I prefer this page format. What should we do @sbrl?

File: pages/common/lynx.md (hunk 1 above)
Comment (alimalek71) [reply pages/common/lynx.md:24]: Ops, nice catch yes of course :+1:

**Comment by navarroaxel** [comment]
@alimalek71 question: if we merge the PR #4764, do you know how to merge from upstream resolving the conflicts and include your examples to the `lynx` page?

**Comment by alimalek71** [comment]
> @alimalek71 question: if we merge the PR #4764, do you know how to merge from upstream resolving the conflicts and include your examples to the `lynx` page?

Yes, I know.

**Comment by navarroaxel** [comment]
@alimalek71 Could you merge the merge Lynx page in this PR?

**Comment by alimalek71** [comment]
Yes, I will start working on it soon

**Comment by alimalek71** [comment]
All conflicts resolved successfully, sir

**Comment by navarroaxel** [comment]
@alimalek71 Looks like the merge is not ok. Which command did you use?

**Comment by alimalek71** [comment]
> @alimalek71 Looks like the merge is not ok. Which command did you use?

I use rebase! if it's not okay I can revert it and use merging solotion

**Comment by navarroaxel** [comment]
@alimalek71 Please because all the commit and changes now are in the diff of this PR

**Comment by alimalek71** [comment]
I assume it is okay now :) plz recheck it

**Comment by alimalek71** [comment]
@alimalek71 requested a review from @navarroaxel

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Please @alimalek71 leave some examples with the hotkeys to use this app, you can add up to 8 examples.

**Review by navarroaxel — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]

**Comment by alimalek71** [comment]
@navarroaxel @schneiderl Thanks you a lot and I am sorry for not being reachable for a long time :heart:

**Pull request merged**